### DISCUSSION TITLE
iTwin.js support for M1 Macbook OSX

### DISCUSSION THREAD

**Discussion by PeterJBassett** [iTwin/itwinjs-core#3470] [Q&A]
Hello, a question, is there a plan for the itwin imodel.js libraries to support M1 Macbook OSX? 
Do you know what the timescales are for this?

**Comment by Carl-Hinkle** [comment]
Hi Peter, we're starting to work on this very soon. We'll post an update as soon as we have a time estimate.

cc: @tcobbs-bentley @toddsouthenbentley

**Comment by PeterJBassett** [comment]
Ok, thank you for your reply

**Reply by calebmshafer** [reply]
Hey @PeterJBassett, just created an issue to track this. Feel free to subscribe for updates over there, https://github.com/iTwin/itwinjs-core/issues/3484.
